admin/python/lsst/qserv/admin/partConfig.py

```diff
@@ -184,6 +184,8 @@ def cssTableOptions(self):
             options['subChunks'] = bool(int(self.get('part.subChunks', 1)))
             options['dirDb'] = self['dirDb']
             options['dirTable'] = self['dirTable']
+            options['lockInMem'] = 'lockInMem' in self ? self['lockInMem'] : False
+            options['scanSpeed'] = 'scanSpeed' in self ? self['scanSpeed'] : 0
```

Comment:
I'd probably also created separate "group" in config file for the shared scan-related options, so the code would be something like:

    options['lockInMem'] = int(self.get('sharedScan.lockInMem', False))
    options['scanSpeed'] = int(self.get('sharedScan.scanSpeed', 0))

and config file would look like:

    sharedScan = {
        lockInMem = 1
        scanSpeed = 3
    }